packages/compass-export-to-language/src/components/editor/editor.jsx

```diff
@@ -34,34 +24,24 @@ class Editor extends PureComponent {
   }
 
   render() {
-    const OPTIONS = {
-      tabSize: 2,
-      fontSize: 11,
-      minLines: 5,
-      maxLines: Infinity,
-      showGutter: true,
-      readOnly: true,
-      highlightActiveLine: false,
-      highlightGutterLine: false,
-      useWorker: false
-    };
-
     const queryStyle = classnames(styles.editor);
 
     return (
       <div className={queryStyle}>
-        <AceEditor
+        <Editor
+          variant={EditorVariant.Generic}
           mode={this.props.language}
-          defaultValue=""
-          value={this.props.value}
-          theme="mongodb"
-          width="100%"
-          editorProps={{$blockScrolling: Infinity}}
-          setOptions={OPTIONS}
+          text={this.props.value}
```

Comment:
Since we're removing the `defaultValue` here, should we default to passing `defaultValue=""}` in the Editor in `editor.tsx`?